React/Fabric/Mounting/ComponentViews/Switch/RCTSwitchComponentView.mm

```diff
@@ -34,9 +34,16 @@ - (instancetype)initWithFrame:(CGRect)frame
 
     _switchView = [[RCTUISwitch alloc] initWithFrame:self.bounds]; // [macOS]
 
+#if !TARGET_OS_OSX // [macOS]
     [_switchView addTarget:self action:@selector(onChange:) forControlEvents:UIControlEventValueChanged];
+#endif // [macOS]
 
+#if !TARGET_OS_OSX // [macOS]
     self.contentView = _switchView;
+#else // [macOS
+    self.contentView = [[RCTUIView alloc] initWithFrame:frame];
+    [self.contentView addSubview:_switchView];
+#endif // macOS]
```

Comment:
do we need to implement `contextView` differently in our shim to avoid this kind of ifdef?

Reply:
I've added a task to revisit this change once the Fabric branch is building, will be much easier to test this way.

Reply:
I see. So the goal right now isn't parity with the paper implementation but just get stuff building. Got it!